pino.js

```diff
@@ -48,21 +48,27 @@ function defineLevelsProperty (onObject) {
     },
     enumerable: true
   })
-  Object.defineProperty(onObject.levels.values, 'silent', {value: 100})
-  Object.defineProperty(onObject.levels.labels, '100', {value: 'silent'})
+  Object.defineProperty(onObject.levels.values, 'silent', {value: Number.POSITIVE_INFINITY})
+  Object.defineProperty(onObject.levels.labels, Number.POSITIVE_INFINITY.toString(), {value: 'silent'})
```

Comment:
no need to convert to string, adding a numerical key to an object turns it into a string anyway